packages/shared/src/components/RichTextEditor/RichTextEditor.tsx

```diff
@@ -5,6 +5,7 @@ import React, { useEffect, useState } from 'react';
 import { RichTextEditor as FondueRichTextEditor } from '@frontify/fondue';
```

Comment:
Button Working, Link not

https://www.loom.com/share/202a5eb8ff2e4788b46e2f6c3ada30b5?sid=47212c60-16ef-48d5-a1ca-5bfade2d8a0b

Reply:
i could not reproduce this with the new zIndices. Can you check again?